src/cmd.js

```diff
@@ -1,117 +1,197 @@
 #!/usr/bin/env node
 
-import minimist from 'minimist'
+
+import fs from 'fs'
```

Comment:
Is this just a performance optimization? The cost of requiring pure functions appears to be very, very low.